### DISCUSSION TITLE
Unable to run "java -jar posters-demo-store-<version>.jar "         # downloaded

### DISCUSSION THREAD

**Discussion by pranathichaluvadi10** [Xceptance/XLT#511] [General]
Hi,

I am unable to run java -jar posters-demo-store-<version>.jar          # downloaded

I am getting below error
![posters](https://github.com/user-attachments/assets/eab7b365-9518-43fb-bc37-2ccd33d49e3b)

**Comment by rschwietzke** [comment]
Posters is no longer part of XLT and also not part of the test suite.  Posters is here: https://github.com/Xceptance/posters-demo-store

Which documentation did you follow?

**Reply by pranathichaluvadi10** [reply]
I am following below URL.
https://docs.xceptance.com/xlt/load-testing/quick-start/30-demo-test-suite/


https://github.com/Xceptance/posters-simple-loadtest-suite

**Reply by rschwietzke** [reply]
The doc here: https://docs.xceptance.com/xlt/load-testing/quick-start/20-demo-application/ talks about the posters application and about the test suite, but you try to start the demo store in the test suite. That is not what the doc says.

**Reply by pranathichaluvadi10** [reply]
I have followed the above doc.

when I am trying to build "mvn clean package"

I am getting below error 
![mvn clean package](https://github.com/user-attachments/assets/10ef1c15-6afa-4601-a827-e3160080b19a)

**Reply by js-xc** [reply]
As the [doc](https://docs.xceptance.com/xlt/load-testing/quick-start/20-demo-application/) says, you could either run the prepackaged jar you downloaded, which is what I see in your first screenshot, or build the project with mvn, which is your second screenshot. What exactly are you trying to do? Can you see the jar file `posters-demo-store-2.4.0.jar` in the directory you are trying to start it from?

**Reply by pranathichaluvadi10** [reply]
No I am unable to see posters-demo-store-2.4.0.jar in the directory
![posters-demo-store-2 4 0 jar file](https://github.com/user-attachments/assets/fcf198b8-54b2-49db-a46c-8c717e219202)

**Reply by js-xc** [reply]
So do you want to build the jar yourself for some reason? Otherwise it might be the easiest solution to just [download the prepackaged jar](https://github.com/Xceptance/posters-demo-store/releases) and run it.

**Reply by pranathichaluvadi10** [reply]
I have downloaded the jar, what is the next step? 

do I need to run "mvn clean package"? if yes, I am facing below error.
![error](https://github.com/user-attachments/assets/13cf2ee7-ac27-4088-91ce-015e8077a333)

**Reply by js-xc** [reply]
We already built the jar for you, so no need to build anything yourself. Just navigate to the folder you downloaded the jar to, and run `java -jar posters-demo-store-<version>.jar`. Please check the [documentation](https://xltdoc.xceptance.com/xlt/load-testing/quick-start/20-demo-application/#running-posters-as-a-console-application) for more info.
